src/components/calcite-accordion/calcite-accordion.tsx

```diff
@@ -74,14 +84,14 @@ export class CalciteAccordion {
     if (!selectionMode.includes(this.selectionMode)) this.selectionMode = "multi";
   }
 
-  componentDidLoad() {
+  componentDidLoad(): void {
```

Comment:
Randomly commenting on this PR: all the Stencil lifecycle methods are `async` (see [doc](https://stenciljs.com/docs/component-lifecycle#async-lifecycle-methods)), so it might be more accurate to explicit it by using declarations as i:
```ts
async componentDidLoad(): Promise<void>
```

Reply:
They're actually `void | Promise<void>`, so either will work. Lifecycle methods are optional now too, so the type can be left off.